### PR TITLE
updating text on the PLSS page

### PR DESCRIPTION
**Pull request by mheagin**
various changes to text and a couple of links added and changed to the current PLSS web page, to reflect current information.
------------------------

### CODE REVIEW

File: data/sgid-cadastre/plss/index.html [hunk 1]
```diff
@@ -49,15 +49,15 @@
...
+      <p>Survey Control added to the PLSS Fabric is provided by the BLM Surveyors, County Surveyors, County Contract Surveyors and Private Surveyors throughout the State. Survey Control can be added or viewed using the <a href="http://mapserv.utah.gov/PLSS">>AGRC PLSS Web Application.</a></p>
+       <p><strong><em>Important Notes for PLSS Fabric Version 1.0:</em>
+      </strong></p>
     <ul>
-      <li>The Fabric works best with ArcMap 10.3.1 with Dec 15, <a href="http://support.esri.com/en/downloads/patches-servicepacks/view/productid/66/metaid/2252">2015 Patch</a>. Improvements will be in ArcMap 10.4.1.</li>
+      <li>The Fabric works best with ArcMap 10.4.1. Improvements will be in ArcMap 10.5 coming out in 2017.</li>
```
Comment (steveoh) [inline data/sgid-cadastre/plss/index.html]: 10.5 is already out

File: data/sgid-cadastre/plss/index.html [hunk 2]
```diff
@@ -30,15 +30,15 @@
 <p>
   <strong>Data Type: GIS Data Layer</strong>
   <br />
-  <strong>Steward(s): AGRC and BLM</strong>
+  <strong>Steward(s): AGRC BLM and SITLA</strong>
```
Comment (steveoh) [inline data/sgid-cadastre/plss/index.html]: how about some comma's

Comment (mheagin) [inline data/sgid-cadastre/plss/index.html]: comma's will work

File: data/sgid-cadastre/plss/index.html (hunk 1 above)
Comment (mheagin) [inline data/sgid-cadastre/plss/index.html]: I thought just the beta version is out or prerelease.....aka "will you test this for us"
leave as is for now.

**Pull request merged**